modules/vulnerability-management20-report-review-create.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-vulnerabilities.adoc
+:_content-type: PROCEDURE
+[id="vulnerability-management20-report-review-create_{context}"]
+= Reviewing report configuration and creating
+
+[role="_abstract"]
+You can review the configuration of your vulnerability report before creating it.
+
+.Procedure
+. In the *Review and create* section, verify that the report parameters, delivery destination, schedule, and format is correct. To make any changes, click *Back* to go to the previous section and edit the fields that you want changed.
+. Perform one of the following actions:
+* Click *Create* to create the report and save it.
+* Click *Create and send* to create the report and send it to the notifiers and distribution list that you configured.
```

Comment:
This should be removed. It was in the previous mocks, but we decided not to support the "Create and send" option.